mkosi/qemu.py

```diff
@@ -444,6 +460,16 @@ def run_qemu(args: MkosiArgs, config: MkosiConfig, qemu_device_fds: Mapping[Qemu
         *shm,
     ]
 
+    if have_kvm and config.architecture.is_native():
```

Comment:
Ah, `have_kvm` is reused here. I think I would still prefer to move the `config.qemu_kvm != ConfigFeature.disabled` check here to make this more local and so as to not have to remember that this contains a check whether this is enabled as well and not just a feature support.